SECURITY.md

```diff
@@ -0,0 +1,8 @@
+
+# Security Policy
+
+## Reporting a Vulnerability
+
+At SecretHub, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.
+
+If you discover a vulnerability, we would like to know about it so we can take steps to address it as quickly as possible. We would like to ask you to help us better protect our clients and our systems. Please checkout the [responsible disclosure policy](https://secrethub.io/security/responsible-disclosure/).
```

Comment:
*check out

Reply:
Thnx! That's what you get when you're typing `git checkout ...` all day :sweat_smile: